src/test/scala/nl/knaw/dans/easy/schema/CrosswalkerSpec.scala

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (C) 2015-2017 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.schema
+
+import nl.knaw.dans.pf.language.ddm.api.Ddm2EmdCrosswalk
+import nl.knaw.dans.pf.language.emd.binding.EmdMarshaller
+import nl.knaw.dans.pf.language.emd.types.ApplicationSpecific.MetadataFormat
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.reflect.io.File
+import scala.xml.{Elem, XML}
+
+class CrosswalkerSpec extends FlatSpec with Matchers {
+
+  "crosswalker" should "convert ddm/example1.xml example1 to EMD" in {
+    toEMD(File("src/main/assembly/dist/docs/examples/ddm/example1.xml")) shouldBe
+      s"""|<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
But where to put it? Under docs/examples/ddm or test/resources? After all these tests are intended to document DDM to EMD conversion (point two of '... it will' ;-), let's not send external developers 'van het kastje naar de muur'. That said, I should rename the test class. 

May be we should dress down to a minimal validating example though we don't validate. Opinions please.

See also the [packaging](https://easy.dans.knaw.nl/schemas/docs/sword-v1-packaging.html) document. Once these tests mature I think we should add a link in that document. But reaching the matured state is beyond the scope of the jira issue.